src/components/NotFoundPage.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+export default class NotFoundPage extends Component {
+  componentDidMount() {}
```

Comment:
whoops.  This could be a functional component.  Didn't put much thought into it.